downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/PathwaySummationMappingFile.java

```diff
@@ -0,0 +1,59 @@
+package org.reactome.release.downloadDirectory;
+
+import java.io.File;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.nio.file.StandardOpenOption;
+import java.util.Collection;
+import java.util.HashSet;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+
+public class PathwaySummationMappingFile {
+	private static final Logger logger = LogManager.getLogger();
+	
+	public static void execute(MySQLAdaptor dba, String releaseNumber) throws Exception {
+		logger.info("Running PathwaySummationMappingFile");
+		// Get all Pathway instances
+		Collection<GKInstance> pathwayInstances = dba.fetchInstancesByClass(ReactomeJavaConstants.Pathway);
+		HashSet<String> rowHash = new HashSet<String>();
+		//Create file
+		logger.info("Generating pathway2summation.txt file...");
+		PrintWriter pathwaySummationFilename = new PrintWriter("pathway2summation.txt");
```

Comment:
Is the PrintWriter used?

Reply:
I believe it generates the file

Reply:
In my IDE, it showed up as an unused variable.  Maybe it was leftover from a change?